### ISSUE TITLE
minitft_featherwing buttons always true when power source not PC

### ISSUE THREAD

**Issue by hexthat** [adafruit/Adafruit_CircuitPython_FeatherWing#40]
```
"""
featherM4 & mini TFT joypad featherwing tester
"""
import board
import neopixel
import terminalio
import time
from adafruit_featherwing import minitft_featherwing
from adafruit_display_text import label
import gc

pixel = neopixel.NeoPixel(board.NEOPIXEL, 1)
pixel[0] = (0, 0, 0)
gc.collect()
print(gc.mem_free())

wing = minitft_featherwing.MiniTFTFeatherWing()

def getbuttons():
    print(wing.buttons)  # added just to see what is going on
    buttons = wing.buttons
    if buttons.right:
        RIGHTtext = "RIGHT "
    else:
        RIGHTtext = ""

    if buttons.down:
        DOWNtext = "DOWN "
    else:
        DOWNtext = ""

    if buttons.left:
        LEFTtext = "LEFT "
    else:
        LEFTtext = ""

    if buttons.up:
        UPtext = "UP "
    else:
        UPtext = ""

    if buttons.a:
        Atext = "A "
    else:
        Atext = ""

    if buttons.b:
        Btext = "B "
    else:
        Btext = ""

    if buttons.select:
        SELtext = "SEL "
    else:
        SELtext = " "

    texts = RIGHTtext + DOWNtext + LEFTtext + UPtext + Atext + Btext + SELtext
    return texts

time.sleep(0.5)

while True:
    text_area = label.Label(terminalio.FONT, text=getbuttons())
    text_area.x = 45
    text_area.y = 38
    wing.display.show(text_area)
```

code tested with

**Comment by makermelissa** [comment]
I need to investigate if this is a seesaw pull-ups issue or something else.

**Comment by makermelissa** [comment]
I noticed the Neopixel on my Feather M4 goes red when it does this. Also if I plugged into non-data USB power and then plug into computer USB, it is still red and behaves this way. This might go deeper and possibly be a CP bug. I'll connect a debugger to my Feather (It's modded) and look into that.

**Comment by hexthat** [comment]
mine stays green when i remove code for neopixel to be off

```
"""
featherM4 & mini TFT joypad featherwing demo
"""
import board
import terminalio
import time
from adafruit_featherwing import minitft_featherwing
from adafruit_display_text import label
import gc

gc.collect()
print(gc.mem_free())

wing = minitft_featherwing.MiniTFTFeatherWing()

def getbuttons():
    print(wing.buttons)
    buttons = wing.buttons
    if buttons.right:
        RIGHTtext = "RIGHT "
    else:
        RIGHTtext = ""

    if buttons.down:
        DOWNtext = "DOWN "
    else:
        DOWNtext = ""

    if buttons.left:
        LEFTtext = "LEFT "
    else:
        LEFTtext = ""

    if buttons.up:
        UPtext = "UP "
    else:
        UPtext = ""

    if buttons.a:
        Atext = "A "
    else:
        Atext = ""

    if buttons.b:
        Btext = "B "
    else:
        Btext = ""

    if buttons.select:
        SELtext = "SEL "
    else:
        SELtext = " "

    texts = (RIGHTtext + DOWNtext + LEFTtext + UPtext + Atext + Btext + SELtext)
    return texts

time.sleep(0.5)

while True:
    text_area = label.Label(terminalio.FONT, text=getbuttons())
    text_area.x = 45
    text_area.y = 38
    wing.display.show(text_area)
```

**Comment by makermelissa** [comment]
Ok thanks @hexthat, I'll throw the latest beta on there. mine might be a wee bit behind.

**Comment by makermelissa** [comment]
Nope, still doing the red thing with Beta 7 for me.

**Comment by hexthat** [comment]
that is strange, why would they be different between boards?

**Comment by makermelissa** [comment]
Also, I'm using the simpletest since the behavior seems to be there too.

**Comment by hexthat** [comment]
okay ill try that

**Comment by hexthat** [comment]
neopixel still green for me, even when plugged into power source not PC

dargh im gana put older cp on it then try new beta 7 again

**Comment by makermelissa** [comment]
Oh and I'm not using your SPI modification. I'm using it as it is in the repo.

**Comment by makermelissa** [comment]
FYI, I just tried on a different feather and it's green.

**Comment by hexthat** [comment]
oh i was about to think it was cause i changed the CS and DC pins, good to know that wasnt it

**Comment by makermelissa** [comment]
Fixed via #39.
